server/channels/app/onboarding.go

```diff
@@ -27,7 +27,7 @@ func (a *App) markAdminOnboardingComplete(c request.CTX) *model.AppError {
 	return nil
 }
 
-func (a *App) CompleteOnboarding(c request.CTX, request *model.CompleteOnboardingRequest) *model.AppError {
+func (a *App) CompleteOnboarding(c request.CTX, request *model.CompleteOnboardingRequest, r *http.Request) *model.AppError {
```

Comment:
Another potential problem here (not necessarily yours, but symptomatic of something bigger): we should haven't to pass a `request.CTX` and `*http.Request` so we can build /another/ context object for plugins. Something is off here and I need to figure out what to recommend.